packages/integrations/cloudflare/src/index.ts

```diff
@@ -1,18 +1,26 @@
 import { createRedirectsFromAstroRoutes } from '@astrojs/underscore-redirects';
+import type { IncomingRequestCfProperties } from '@cloudflare/workers-types/experimental';
 import type { AstroAdapter, AstroConfig, AstroIntegration, RouteData } from 'astro';
 import esbuild from 'esbuild';
 import * as fs from 'node:fs';
 import * as os from 'node:os';
 import { sep } from 'node:path';
 import { fileURLToPath, pathToFileURL } from 'node:url';
 import glob from 'tiny-glob';
+import { CFVars } from './CFVars.js';
 
 export type { AdvancedRuntime } from './server.advanced';
 export type { DirectoryRuntime } from './server.directory';
 
 type Options = {
 	mode: 'directory' | 'advanced';
 	functionPerRoute?: boolean;
+	/**
+	 * 'off': current behaviour (wrangler is needed)
+	 * 'local': use a static req.cf object, and env vars defined in wrangler.toml & .dev.vars (astro dev is enough)
+	 * 'remote': use a dynamic real-live req.cf object, and env vars defined in wrangler.toml & .dev.vars (astro dev is enough)
```

Comment:
I'm not sure what "astro dev is enough" means here. Could you briefly mention the alternative in this typedoc?

Reply:
For context, previously one had to either use wrangler in "proxy-mode" or build the Astro project and point to the dist folder. The phrase "astro dev is enough" signifies that running astro dev now provides runtime information, a feature previously exclusive to wrangler.

This comment is intended solely for contributors and code-reviewers; detailed documentation will be added in the second phase of this topic.